### ISSUE TITLE
Put together a design brief template

### ISSUE THREAD

**Issue by cassiemc** [MozillaFoundation/plan#300]
Some folks have requested a sample design brief w/ standardized questions. I put something together here: https://docs.google.com/a/mozilla.com/document/d/1E6lhyYjty_eNUo2xVAKcde6h7Di2zI_iShcIcmlz9aw/edit?usp=sharing

Thoughts @iamjessklein @xmatthewx @flukeout @vazquez @sabrinang @davidascher – anything you would take away or add?

**Comment by xmatthewx** [comment]
I added some comments and an alternative narrative below. Is this for in-house MoFo projects?

**Comment by iamjessklein** [comment]
:+1: me too

**Comment by cassiemc** [comment]
You guys are amazing! Love all the suggestions and the cleaned up narrative mw. Yes it's for internal use. I'd like to make it optional/helpful as opposed to an upfront requirement, though it should be helpful to designers as we enter kickoff meetings. I think I'll clean this up then file to add it to build.webmaker.org, and/or put it in the resources drive.

**Comment by cassiemc** [comment]
After conversation with @flukeout yesterday I wanted to bring this bug back from the dead. I've revised some questions and added some context – Luke could you R? and gimme some feedback / ideas? https://docs.google.com/document/d/1E6lhyYjty_eNUo2xVAKcde6h7Di2zI_iShcIcmlz9aw/edit#

**Comment by flukeout** [comment]
Great stuff in the doc! I think the second set of questions does a good job of getting at the core of the design problem. Here are some themes I'm interested in...

**Benchmarking for Function & Aesthetics**
When designing aspects of the Webmaker app, it was really useful to look at how various UX problems have been solved by other, successful platforms. I think we should incorporate this type of research into all of our design activities. For example, before starting a design, we could find three examples that we could model after and discuss them with the team.

**Style Guidelines**
At Mozilla, we've got a lot of different properties, many of which don't exactly match. When we're designing something new, let's be intentional about which style guidelines we're following and which of our may properties our design should mesh with. This will help us make decisions and ensure the work we're doing will reinforce the language we've already established.

If we notice that our designs are moving away from an established direction, we can be intentional about that too. Is this a one-off, or is this an experiment in evolving our overall language.

**Measuring Success**
We've had some success recently with measuring the effectiveness of our design work through Usertesting.com and Laura's research, what are some other ideas (services, tools or methods) for measuring what we do?
* Web analytics
* Usertesting.com
* ...?

The more we're able to know about the performance of the work we've done in the past, the better we can be at designing for the future.

**User Stories**
User stories are a granular way to define what a product is supposed to achieve, [you can read more here](https://www.mountaingoatsoftware.com/agile/user-stories). Basically, you can frame product features this kind of format...
* As a <type of user>, I want <some goal> so that <some reason>.
Before we design, we can list out the user stories that we want to satisfy through our designs. This can help guide our work and also give us something to reflect on (via metrics if we have them) when we're finished.

**Comment by xmatthewx** [comment]
Here's my proposal: 
 1. New projects added to /plan/ that are flagged as `needs-design` get the checklist below added to it. 
 2. Project driver can 
  - cross out items that are not relevant ~~like so~~
  - add a written comment and check off items
 3. Remaining items can be discussed and checked off during kickoff meetings
 4. We can improve the list each time we use it (But!! keep it nimble and low-friction so we actually want to use this process)

Ultimately, I think it is on us designers to arrive with these questions and make sure they get answered. We can guide and help build the practice so it becomes second nature. 


#### Project Brief

 - [ ] What problem are you trying to solve?
 - [ ] Who has this problem? Do you have research or user stories?
 - [ ] When do you plan to launch your solution? How long will it remain active? Describe opportunities, constraints, phases.
 - [ ] What features does an MVP solution include? How will your audience use it?
 - [ ] What is the value to your audience? How many people will be served?
 - [ ] How will you market and distribute the solution? What channels will drive the audience to it? (ex: FF Tiles, Facebook Ads, Hive Network)
 - [ ] How will you measure success? What are the needs and opportunities for...
   - [ ] user testing? 
   - [ ] analytics?
   - [ ] field research?
 - [ ] Who else in the field have addressed similar problems for similar audiences? What benchmarks serve as good reference?
 - [ ] What Mozilla properties or style guides should inform the design? (ex: Material Design ) 
 - [ ] If you overheard someone describing their experience with your project, what would you love to hear? Include 2-4 adjectives that summarize the spirit of the experience.

#### Details

 - [ ] Project URL, repo
 - [ ] Required regions for localization
 - [ ] Visual content (existing assets & requirements)
 - [ ] Written content (existing assets & requirements)
 - [ ] Other requirements or restrictions (ex: consent to terms)

**Comment by cassiemc** [comment]
Argh, I thought I responded to this already. Anywho, love the idea of the checklist @xmatthewx. What about a ```needs design brief``` label instead? ```Needs-design``` is generally used for resourcing purposes. I like this process and just want to add a couple points into the Brief document re. user stories and success metrics, as per @flukeout's comment above.

Also pinging @sabrinang @vazquez and @lovegushwa for review.

Here's the simple diagram I've been hacking to help illustrate the process:

![design-brief-process-lowres-half](https://cloud.githubusercontent.com/assets/1186326/7993885/54d6c0d2-0ad7-11e5-810c-ffaa9946f970.png)

**Comment by cassiemc** [comment]
Here's an updated graphic:
![design brief process-v3](https://cloud.githubusercontent.com/assets/1186326/7998086/7504dea6-0b05-11e5-907e-c84655650f14.png)

Wondering if @OpenMatt @edrushka @hannahkane @ldecoursy @secretrobotron @HPaulJohnson have any feedback on this proposed process, ahead of our meeting to discuss next week? All is game for critique, whether nitpicky language edits or big picture process.

**Comment by cassiemc** [comment]
To do to close this ticket:
- [ ] Final edits to the design brief
- [ ] Link to the design brief from build.webmaker.org
- [ ] Get feedback from PMs on the process
- [ ] Introduce process to the engineers
- [ ] Socialize with all@mofo (email)

**Comment by xmatthewx** [comment]
@cassiemc When I posted the checklist above (https://github.com/MozillaFoundation/plan/issues/300#issuecomment-106087019) from the stuff I drafted in the google doc, I integrated feedback from Luke. Probably not represented back in the google doc.

**Comment by xmatthewx** [comment]
The "new" form at https://build.webmaker.org/add generates a new issue and RACI. We can have it add this checklist as well, maybe as a comment. Might be more PM-friendly than a brief filed away in google drive that requires copy-paste or save as...

**Comment by cassiemc** [comment]
Except no one uses that add link, apparently.

**Comment by cassiemc** [comment]
This is done. Accessible at build.webmaker.org/design
